plugins/tiddlywiki/webdav/html-div-tiddler.tid

```diff
@@ -0,0 +1,9 @@
+title: $:/core/templates/html-div-tiddler
+
+<!--
+
+This template is used for saving tiddlers as an HTML DIV tag with attributes representing the tiddler fields. This version includes the tiddler changecount as the field `revision`.
+
+-->`<div`<$fields exclude='text revision bag' template=' $name$="$encoded_value$"'></$fields>` revision="`<<changecount>>`" bag="default">
```

Comment:
server specific fields need to be built dynamically at load time. .... They are _not_ saved in the .tid file!

Reply:
server fields IMO should not saved in the "Download" single file version

Reply:
Sorry I'm still not failiar with what server fields are etc - I'm seeing some issues with the saver but I don't understand yet what's wrong.

Reply:
I think I know which problems you have. I think I had the same problems when I was messing around with the dat-adaptor, which wasn't released, because the dat-api needed was removed. ... 

I'll have a closer look, too. ... But this weekend will be limited time, because of "family business" :)